grails-app/conf/Config.groovy

```diff
@@ -291,14 +291,10 @@ marvl {
 // WFS indexed file info
 indexedFile.fileSizeColumnName = "size"
 
-// Something else is adding an appender (what?!) - get rid of it, else we end up
-// with duplicate logs in production.
-org.apache.log4j.Logger.rootLogger.removeAllAppenders()
```

Comment:
You mean with appenders removed we were still getting double logs?  In any case, I just checked on edge and we are *not* getting double logs currently, whoot!